service/systemd/service.go

```diff
@@ -221,6 +224,10 @@ func (s *Service) check() (bool, error) {
 	conf, err := s.readConf()
 	if err != nil {
 		return false, errors.Trace(err)
+	} else if (err == nil && true == reflect.DeepEqual(conf, common.Conf{})) {
```

Comment:
I don't think you need this change else if clause - since `readConf` will return an empty conf for the fallback, the `return reflect.DeepEqual(normalConf, conf), nil` at the end will do the right thing, won't it?